dff/script/core/types.py

```diff
@@ -40,39 +40,56 @@
 # TODO: change example
 
 
-# TODO: add description for each stage of ActorStage
 class ActorStage(Enum):
     """
     The class which holds keys for the handlers. These keys are used
-    for the actions of :py:class:`~dff.script.Actor`.
+    for the actions of :py:class:`.Actor`. Each stage represents
+    a specific step in the conversation flow. Here is a brief description
+    of each stage.
     """
 
-    #: This stage is used for the context initializing.
+    #: This stage is used for the context initialization.
+    #: It involves setting up the conversation context.
```

Comment:
Why is a different docstring style used here?
The rest of the library uses `"""""`.